front-end/src/components/Admin/Grid/index.js

```diff
@@ -130,6 +130,17 @@ class Grid extends Component {
   }
 
   componentDidMount () {
+    const sortCells = document.getElementsByClassName('tabulator-sortable');
+    const table = this.tabulatorTable;
+    table.addEventListener('click', function (event) {
```

Comment:
Please check if tabulator doesn't have a setting to achieve this. This code is written not in react-way, although it is fine for pure JS or jQuery style.

Reply:
ok, i'll check this